modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/cellproperties/v0.1.0.json

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/cellproperties/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:CellPropertiesShape",
+      "@type": "sh:NodeShape",
+      "targetClass": "nsg:CellProperties",
```

Comment:
CellProperty is a bit too broad. What properties will be find in the files ? Can this type be better named?

Reply:
This is a file that contains all the properties for the cells.
It is broad on purpose because we want to be able to store any properties there.

Reply:
Understood. Can it be nsg:CircuitCellProperties ?